admin_guide/limits.adoc

```diff
@@ -117,10 +117,19 @@ spec:
       max:
         openshift.io/image-tags: 20 <2>
         openshift.io/images: 30 <3>
+    - type: "Pod"
+      max:
+        cpu: "2" <4>
+      	memory: "1Gi" <5>
+      	ephemeral-storage: "1Gi" <6>
+      
 ----
 <1> The maximum size of an image that can be pushed to an internal registry.
 <2> The maximum number of unique image tags per image stream's spec.
 <3> The maximum number of unique image references per image stream's status.
+<4> The maximum amount of CPU that a pod can request on a node across all containers
+<5> The maximum amount of memory that a pod can request on a node across all containers
+<6> The maximum amount of ephemeral storage that a pod can request on a node across all containers
```

Comment:
Same here, need to make sure this is identified as tech preview

Reply:
Yes.